**AlphaFold**

Github Repository: [_https://github.com/Merylin-O/Hackbio-Cancer-Internship_](https://github.com/Merylin-O/Hackbio-Cancer-Internship)

 

**A BRIEF INTRODUCTION ON ALPHAFOLD**

The field of molecular biology has undergone a remarkable change with the advancement of technologies, including the introduction of different tools that have revolutionized the field of genomics, proteomics, and structural biology (Forero & Chand & 2023; Koff _et al._, 2023). One of the most significant breakthroughs is the introduction of several bioinformatics tools that have enabled the accurate interpretation and analysis of genomic data, which is important in various applications, such as drug discovery, disease diagnosis, disease prevention, and personalized medicine. AlphaFold, an artificial intelligence program developed by DeepMind, is one of the most remarkable tools in this field (AlQuraishi, 2019). It was developed as an entry for the 13th Critical Assessment of Protein Structure Prediction (CASP3) competition.

 

**PROTEIN STRUCTURE PREDICTION**Proteins are complex structures made of amino acid residues that are joined together by peptide bonds. Their 3D structure allows them to perform different functions. To achieve this structure, proteins fold into various conformations. Prediction of protein structures is a significant part and challenge of structural biology. The prediction of protein structures has been ongoing since the 1900s, when the prediction of the secondary structure of proteins began in 1951 by Pauling and Corey (Yang _et al_., 2016). Before the development of several algorithms to improve protein structure prediction, protein structures were predicted using X-ray, nuclear magnetic resonance, and electron cryomicroscopy. However, these methods were not sufficient due to the huge amount spent to predict proteins, the time consumption, and the large number of protein sequences available (Bertoline _et al_., 2023).

 

**EVOLUTION OF ALPHAFOLD**

AlphaFold uses a novel machine learning approach that leverages large datasets of protein structures, incorporating physical and biological knowledge (Jumper _et al_., 2021). The output result gives a highly accurate prediction of the 3D structure of the input protein, which is often indistinguishable from experimental structures (Lupas _et al_., 2021; Marcu _et al_., 2022).

****Since the development of the AlphaFold algorithm, there have been evolutionary changes in the model, including the introduction of AlphaFold 2 and the recent release of AlphaFold 3 (Abramson _et al_., 2024). These updated versions have higher accuracy in predictions. Additionally, the AlphaFold Protein Database has been introduced, containing over 350,000 predicted protein structures across 21 proteomes (Varadi _et al_., 2021). This makes different predicted protein structures readily available.

 

**CONCLUSION**

In conclusion AlphaFold has transformed the field of protein structure prediction, achieving remarkable accuracy and speed. With its ability to predict protein structure at such high scale, it can create new avenues for research. As the tool continues to grow, it will provide breakthroughs in our understanding of genomics, proteomics, and structural biology.

 

 

****

**References**

Abramson, J., Adler, J., Dunger, J., Evans, R., Green, T., Pritzel, A., Ronneberger, O., Willmore, L., Ballard, A. J., Bambrick, J., Bodenstein, S. W., Evans, D. A., Hung, C.-C., O’Neill, M., Reiman, D., Tunyasuvunakool, K., Wu, Z., Žemgulytė, A., Arvaniti, E., & Beattie, C. (2024). Accurate structure prediction of biomolecular interactions with AlphaFold 3. _Nature_, _630_, 1–3. https\://doi.org/10.1038/s41586-024-07487-w

AlQuraishi, M. (2019). AlphaFold at CASP13. _Bioinformatics_, _35_(22), 4862–4865. https\://doi.org/10.1093/bioinformatics/btz422

Bertoline, L. M. F., Lima, A. N., Krieger, J. E., & Teixeira, S. K. (2023). Before and after AlphaFold2: An overview of protein structure prediction. _Frontiers in Bioinformatics_, _3_. https\://doi.org/10.3389/fbinf.2023.1120370

Forero, D. A., & Chand, V. (2023). Methods in molecular biology and genetics: looking to the future. _BMC Research Notes_, _16_(1). https\://doi.org/10.1186/s13104-023-06298-y

Jumper, J., Evans, R., Pritzel, A., Green, T., Figurnov, M., Ronneberger, O., Tunyasuvunakool, K., Bates, R., Žídek, A., Potapenko, A., Bridgland, A., Meyer, C., Kohl, S. A. A., Ballard, A. J., Cowie, A., Romera-Paredes, B., Nikolov, S., Jain, R., Adler, J., & Back, T. (2021). Highly accurate protein structure prediction with alphafold. _Nature_, _596_(7873), 583–589. https\://doi.org/10.1038/s41586-021-03819-2

Koff, W. C., Rino Rappuoli, & Plotkin, S. А. (2023). Historical Advances in Structural and Molecular Biology and How They Impacted Vaccine Development. _Journal of Molecular Biology_, _435_(13), 168113–168113. https\://doi.org/10.1016/j.jmb.2023.168113

Lupas, A. N., Pereira, J., Alva, V., Merino, F., Coles, M., & Hartmann, M. D. (2021). The breakthrough in protein structure prediction. _Biochemical Journal_, _478_(10), 1885–1890. https\://doi.org/10.1042/bcj20200963

Marcu, Ş.-B., Tăbîrcă, S., & Tangney, M. (2022). An Overview of Alphafold’s Breakthrough. _Frontiers in Artificial Intelligence_, _5_. https\://doi.org/10.3389/frai.2022.875587

Varadi, M., Anyango, S., Deshpande, M., Nair, S., Natassia, C., Yordanova, G., Yuan, D., Stroe, O., Wood, G., Laydon, A., Žídek, A., Green, T., Tunyasuvunakool, K., Petersen, S., Jumper, J., Clancy, E., Green, R., Vora, A., Lutfi, M., & Figurnov, M. (2021). AlphaFold Protein Structure Database: massively expanding the structural coverage of protein-sequence space with high-accuracy models. _Nucleic Acids Research_, _50_(D1). https\://doi.org/10.1093/nar/gkab1061

Yang, Y., Gao, J., Wang, J., Heffernan, R., Hanson, J., Paliwal, K., & Zhou, Y. (2016). Sixty-five years of the long march in protein secondary structure prediction: the final stretch? _Briefings in Bioinformatics_, bbw129. https\://doi.org/10.1093/bib/bbw129

 
